core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/SeekableByteArrayInputStream.java

```diff
@@ -42,13 +43,13 @@ public class SeekableByteArrayInputStream extends InputStream {
   @SuppressFBWarnings(value = "VO_VOLATILE_REFERENCE_TO_ARRAY",
       justification = "see explanation above")
   private volatile byte[] buffer;
-  private int cur;
-  private int max;
+  private final AtomicInteger cur;
```

Comment:
In both constructors, you're initializing it to 0. You can just do that here once.

```suggestion
  private final AtomicInteger cur = new AtomicInteger(0);
```